pages/en/network/developer.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: Developing
+---
+
+Developers are a core part of the graph ecosystem. They’re building and querying subgraphs making high-quality data available to users through simple interfaces.
+
+Anyone can build and deploy a subgraph that queries data from a specific part of the blockchain.
+
+Once a subgraph has been deployed to the decentralized network, developers can begin making queries to it using GRT. Subgraphs are available to anyone to query data.
+
+There are 5 steps to a subgraph’s lifecycle.
+
+## Developing
+
+The Graph CLI tool streamlines the process of creating a subgraph. When you initialize your subgraph, all the default files will be created and configured.
+
+When writing your subgraph mappings, you can use the built-APIs or add other libraries as dependencies as long as they are compatible with AssemblyScript.
```

Comment:
Did you mean "built-in APIs"?

Reply:
Yes, sorry about that. Cleaning this up today!